main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+func main() {
+
+	router := mux.NewRouter()
+
+	router.HandleFunc("/pokemons", getPokemons).Methods("GET")
+
+	fmt.Println("Server at 8000")
+	log.Fatal(http.ListenAndServe(":8000", router))
+	fmt.Println("Ejemplo")
+}
+
```

Comment:
Add comments to all methods that it needed for example:

// MethodName - It reads a csv file and return in as a slice struct